File: lotteryticket/src/main/java/com/wuhai/lotteryticket/config/network/APIBaseService.java
package com.wuhai.lotteryticket.config.network;


import com.wuhai.retrofit.retrofit.BaseApi;

/**
 * 作者 wuhai
 *
 * 创建日期 2018/5/16 17:03
 *
 * 描述：创建网络接口
 */
public abstract class APIBaseService {

    public API api = BaseApi.INSTANCE.get(API.class);
}

File: lotteryticketkotlin/src/main/java/com/wuhai/demo/lotteryticketkotlin/config/network/APIBaseService.kt
package com.wuhai.demo.lotteryticketkotlin.config.network

import com.wuhai.retrofit.retrofit.BaseApi

/**
 * 作者 wuhai
 *
 * 创建日期 2018/5/16 17:03
 *
 * 描述：创建网络接口
 */
abstract class APIBaseService {
    @JvmField
    var api = BaseApi.INSTANCE[API::class.java]
}